yt_dlp/extractor/facebook.py

```diff
@@ -481,6 +481,10 @@ def extract_metadata(webpage):
                 r'data-sjs>({.*?ScheduledServerJS.*?})</script>', webpage)]
             post = traverse_obj(post_data, (
                 ..., 'require', ..., ..., ..., '__bbox', 'require', ..., ..., ..., '__bbox', 'result', 'data'), expected_type=dict) or []
+            post_stats = traverse_obj(post, (
+                lambda _, v: v['short_form_video_context']['video']['id'] == video_id and v[
+                    'url'] == f'https://www.facebook.com/reel/{video_id}/'), expected_type=dict) or []
+            post_stats = get_first(post_stats, ('feedback', {dict}), default={})
```

Comment:
These two lines can just be combined to:
```suggestion
            post_stats = traverse_obj(post, (
                lambda _, v: v['short_form_video_context']['video']['id'] == video_id and v[
                    'url'] == f'https://www.facebook.com/reel/{video_id}/', 'feedback', {dict}, any)) or {}
```
Though maybe there is an easier way than checking both the URL and video id.